ci/config.yaml

```diff
@@ -3,3 +3,11 @@ kraken_db:
 ranks:
     # if not in a list, it will mess up
     ["genus", "species"]
+methods:
+  ["kraken2", "metaphlan2"] # "shogun"]
+metrics:
+  ['correlation', 'l2_norm', 'auprc', 'absolute_error']
```

Comment:
I wonder if `correlation` would better be `pearsonr`, as there is also `spearmanr`.

Reply:
done.